src/components/login.tsx

```diff
@@ -52,56 +51,91 @@ export class LoginScreenComponent extends React.Component<LoginProps, LoginState
-            onChangeText={(text) => this.setState({ username: text })}
-            style={Styles.input}
-          />
-        </View>
-
-        <View style={Styles.inputWrapper}>
-          <TextInput
-            placeholder='Password'
-            secureTextEntry={true}
-            onChangeText={(text) => this.setState({ password: text })}
-            style={Styles.input}
-          />
-        </View>
-
-        <View style={Styles.switchContainer}>
-          <Switch
-            value={this.state.rememberMe}
-            onValueChange={(value) => this.setState({rememberMe: value})}
-          />
-
-          <Text>
-            Stay signed in
-          </Text>
-        </View>
-
-        <Text>
-          { this.props.error }
-        </Text>
-
-        <Button title='Login' onPress={this.submit.bind(this)} />
-
-        <Text style={Styles.paragraph} onPress={() => navigate('SignUp')}>
-          Don't have an account? <Text style={Styles.link} >Sign up</Text>
-        </Text>
-      </KeyboardAvoidingView>
+      <Container style={Styles.viewContainer}>
+        <Content>
+          <ImageBackground style={Styles.loginHeader} source={require('../../images/loginHeader.png')}>
+            <Image source={require('../../images/g-logo.png')} style={Styles.loginHeaderImage} />
+
+            <Text style={Styles.loginHeaderTitle}>
+              GENERIC SYSTEMS
+            </Text>
+          </ImageBackground>
+
+          <View style={Styles.loginBody}>
+            <Form style={Styles.form}>
+              <Item style={Styles.formItem} stackedLabel>
+                <Label>{ username ? 'Email' : '' }</Label>
+
+                <Input
+                  style={Styles.formInput}
+                  placeholder='Email'
+                  keyboardType='email-address'
+                  autoCapitalize='none'
+                  autoCorrect={false}
+                  onChangeText={(text) => this.setState({ username: text })}
+                />
+              </Item>
+
+              <Item style={Styles.formItem} stackedLabel>
+                <Label>{ password ? 'Password' : '' }</Label>
+
+                <Input
+                  style={[Styles.formInput, Styles.formInputPass]}
+                  placeholder='Password'
+                  secureTextEntry={true}
+                  onChangeText={(text) => this.setState({ password: text })}
+                />
+
+                <Text style={Styles.formInputLink}>
+                  FORGOT?
+                </Text>
+              </Item>
+
+              <View style={Styles.switchContainer}>
+                <Switch
+                  value={this.state.rememberMe}
+                  onValueChange={(value) => this.setState({rememberMe: value})}
+                />
+
+                <Text style={Styles.switchText}>
+                  Stay signed in
+                </Text>
+              </View>
+            </Form>
+
+            <Text>
+              { this.props.error }
```

Comment:
With the style change i think we should change this error to have the grey font that for instance "Styles.switchText" has. 
<img width="381" alt="screen shot 2018-02-28 at 11 32 41 am" src="https://user-images.githubusercontent.com/7120180/36799390-1ac5ae5e-1c7b-11e8-8200-073d872c7c94.png">